cherrypy/_cperror.py

```diff
@@ -34,6 +34,7 @@
                                               POST should not raise this error
 305      Use Proxy                            Confirm with the user
 307      Temporary Redirect                   Confirm with the user
+308      Moved Permanently                    Confirm with the user
```

Comment:
There's line 42 starting with `or 307 when redirecting POST`, it should probably also mention 308.